_config.yml

```diff
@@ -151,9 +151,10 @@ defaults:
 #   - dest is the destination path within the working tree
 #   - src is a list of glob source paths within the remote repository
 fetch-remote:
-  - repo: "https://github.com/docker/cli"
+  # FIXME(thaJeztah): temporarily using branch from https://github.com/docker/cli/pull/3525 to fix htmlproofer validation
+  - repo: "https://github.com/thaJeztah/cli"
     default_branch: "master"
-    ref: "20.10"
+    ref: "20.10_backport_anchor_tags"
```

Comment:
- This is temporary for this PR, and will be removed before merging (after https://github.com/docker/cli/pull/3525 is merged)